src/components/comments/comment.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  Button, Comment, Popup, Form, Checkbox,
+} from 'semantic-ui-react';
+import '../../styles/profile.scss';
+import { fetchComments } from '../../actions/commentAction';
+import { updateComment } from '../../actions/postUpdateCommentAction';
+import { deleteComment } from '../../actions/deleteComentAction';
+import EditCommentComponent from './editComment';
+import ReplyCommentComponent from './replyComment';
+
+const username = localStorage.getItem('username');
+
+// eslint-disable-next-line react/prefer-stateless-function
+class CommentSingleComment extends React.Component {
+  state = {
+    isUpdatePopUpOpen: false,
+    isDeletePopUpOpen: false,
+    collapsed: true,
+    isReplyPopUpOpen: false,
+  };
+
+  handleCheckbox = (e, { checked }) => this.setState({ collapsed: checked })
+
+  handleUpdateOpen = () => {
+    this.setState({ isUpdatePopUpOpen: true, isDeletePopUpOpen: false });
+  };
+
+  handleReplyOpen = () => {
+    this.setState({ isReplyPopUpOpen: true });
+  };
+
+  handleReplyClose = () => {
+    this.setState({ isReplyPopUpOpen: false });
+  };
+
+  handleUpdateClose = () => {
+    this.setState({ isUpdatePopUpOpen: false, isDeletePopUpOpen: false });
+  };
+
+  handleClose = () => {
+    this.setState({ isUpdatePopUpOpen: false, isDeletePopUpOpen: false });
+  }
+
+  handleDeleteOpen = () => {
+    this.setState({ isUpdatePopUpOpen: false, isDeletePopUpOpen: true });
+  };
+
+  handleDeleteClose = () => {
+    this.setState({ isUpdatePopUpOpen: false, isDeletePopUpOpen: false });
+  };
+
+  handleDelete = (e) => {
+    const {
+      comment, article,
+    } = this.props;
+    const data = { articleSlug: this.props.slug, commentId: comment.id };
+
+    this.props.deleteComment(data);
+  };
+
+  render() {
+    const { isUpdatePopUpOpen, isDeletePopUpOpen, isReplyPopUpOpen, collapsed } = this.state;
+    const { comment } = this.props;
+    if (this.props.comments.isFetched) {
+      return (
+        <Comment>
+          <Comment.Avatar id="avatar" src="https://react.semantic-ui.com/images/avatar/small/matt.jpg" />
+          <Comment.Content>
+            <Comment.Author id="author" as="a">{comment.author}</Comment.Author>
+            <Comment.Metadata id="date">
+              <div>{comment.created_at}</div>
+            </Comment.Metadata>
+            <Comment.Text className="comment-body">{comment.body}</Comment.Text>
+            {/* {' '} */}
+            <Comment.Actions>
+              {comment.author === username ? (
+                <Popup
+                  id="update-comment-body"
+                  trigger={(
+                    <Comment.Action>
+                      <Button icon="pencil alternate" />
+                    </Comment.Action>
+)}
```

Comment:
Check this linting issue... Closing brackets and braces should align with the opening.